packages/edit-site/src/components/global-styles/screen-revisions/style.scss

```diff
@@ -41,13 +41,25 @@
 		// This border serves as a background color in Windows High Contrast mode.
 		border: 4px solid transparent;
 	}
+
 	&.is-selected {
 		border-radius: $radius-block-ui;
 
 		// Only visible in Windows High Contrast mode.
 		outline: 3px solid transparent;
 		outline-offset: -2px;
 
+		color: var(--wp-components-color-accent, var(--wp-admin-theme-color, #007cba));
+		background: rgba(var(--wp-admin-theme-color--rgb), 0.04);
+
+		.edit-site-global-styles-screen-revisions__revision-button {
+			opacity: 1;
```

Comment:
Just in case anyone else wonders while looking at this: the `opacity: 1` rule is required so that the `.components-button:disabled` opacity of `.3` isn't applied here.